saleor/graphql/translations/resolvers.py

```diff
@@ -28,6 +29,9 @@
     shipping_models.ShippingMethod: (
         dataloaders.ShippingMethodTranslationByIdAndLanguageCodeLoader
     ),
+    shipping_interface.ShippingMethodData: (
+        dataloaders.ShippingMethodTranslationByIdAndLanguageCodeLoader
+    ),
```

Comment:
Should we add new tests for that?

Reply:
The tests are already there, we just changed the root type from `ShippingMethod` to `ShippingMethodData`. Commenting out this line results in:

```
TypeError: No dataloader found to <class 'saleor.shipping.interface.ShippingMethodData'>
========================================================================= short test summary info ==========================================================================
FAILED saleor/graphql/checkout/tests/test_checkout.py::test_checkout_selected_shipping_method - AssertionError: [{'message': "No dataloader found to <class 'saleor.shipp...
FAILED saleor/graphql/checkout/tests/test_checkout.py::test_checkout_available_shipping_methods_with_weight_based_shipping_method[0] - AssertionError: [{'message': "No d...
FAILED saleor/graphql/checkout/tests/test_checkout.py::test_checkout_available_shipping_methods_with_weight_based_shipping_method[None] - AssertionError: [{'message': "N...
FAILED saleor/graphql/checkout/tests/test_checkout.py::test_checkout_available_shipping_methods_with_weight_based_shipping_method[2] - AssertionError: [{'message': "No d...
FAILED saleor/graphql/checkout/tests/test_checkout.py::test_checkout_available_shipping_methods_weight_method_with_higher_minimal_weigh - AssertionError: [{'message': "N...
FAILED saleor/graphql/checkout/tests/test_checkout.py::test_checkout_available_shipping_methods[shippingMethods] - AssertionError: [{'message': "No dataloader found to <...
FAILED saleor/graphql/checkout/tests/test_checkout.py::test_checkout_available_shipping_methods_with_price_displayed - AssertionError: [{'message': "No dataloader found ...
FAILED saleor/graphql/checkout/tests/test_checkout.py::test_checkout_shipping_methods_with_price_based_shipping_method_and_discount - AssertionError: [{'message': "No da...
FAILED saleor/graphql/checkout/tests/test_checkout.py::test_checkout_available_shipping_methods[availableShippingMethods] - AssertionError: [{'message': "No dataloader f...
========================================================== 9 failed, 5774 passed, 10 skipped in 63.98s (0:01:03) ===========================================================
```